src/Data/Vector/Mutable/Linear.hs

```diff
@@ -141,6 +241,29 @@ instance Consumable (Vector a) where
 -- # Internal library
 -------------------------------------------------------------------------------
 
+-- | Grows the vector to the closest power of growthFactor to
+-- fit at least n more elements.
+growToFit :: HasCallStack => Int -> Vector a #-> Vector a
+growToFit n vec =
+  capacity vec & \(vec', Unrestricted cap) ->
+    size vec' & \(vec'', Unrestricted s') ->
+      if s' + n <= cap
+      then vec''
+      else
+        let -- Calculate the closest power of growth factor
+            -- larger than required size.
+            newSize =
+              constGrowthFactor
```

Comment:
Add a comment mentioning this is a constant defined in this file above.

Reply:
It feels a bit odd to add a comment like `-- this is defined at the top of this file`. I would say that it is the IDE's responsibility (or `grep`'s). Anyway, a comment wont harm anyone, so I'm happy to put the it there (unless you suggest a better one :)).